core/src/main/java/edu/wpi/grip/core/operations/composite/CascadeClassifierOperation.java

```diff
@@ -0,0 +1,132 @@
+package edu.wpi.grip.core.operations.composite;
+
+import edu.wpi.grip.core.Operation;
+import edu.wpi.grip.core.OperationDescription;
+import edu.wpi.grip.core.sockets.InputSocket;
+import edu.wpi.grip.core.sockets.OutputSocket;
+import edu.wpi.grip.core.sockets.SocketHint;
+import edu.wpi.grip.core.sockets.SocketHints;
+import edu.wpi.grip.core.util.Icon;
+
+import com.google.common.collect.ImmutableList;
+
+import org.bytedeco.javacpp.opencv_core.Mat;
+import org.bytedeco.javacpp.opencv_core.Rect;
+import org.bytedeco.javacpp.opencv_core.RectVector;
+import org.bytedeco.javacpp.opencv_core.Size;
+import org.bytedeco.javacpp.opencv_objdetect.CascadeClassifier;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Operation for identifying parts of an image with a cascade classifier.
+ */
+public class CascadeClassifierOperation implements Operation {
+
+  public static final OperationDescription DESCRIPTION =
+      OperationDescription.builder()
+          .name("Cascade Classifier")
+          .summary("Runs a cascade classifier on an image")
+          .icon(Icon.iconStream("opencv"))
```

Comment:
> It's basically a one-to-one mapping of an OpenCV function

All the other OpenCV operations use the OpenCV icon. So this will use the OpenCV icon, too.